Src/Plugins/Tests.Security/Validators/SEC101_050.IdentifiableNpmAuthorTokenValidatorTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Helpers;
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
+{
+    /// <summary>
+    /// Testing SEC101/050.IdentifiableNpmAuthorTokenValidator
+    /// </summary>
+    public class IdentifiableNpmAuthorTokenValidatorTests
+    {
+        [Fact]
+        public void IdentifiableNpmAuthorTokenValidatorTests_MockHttpTests()
```

Comment:
>Tests [](http://example.com/codeflow?start=55&length=5)

nit: remove Tests from this.
The idea that we follow:
ClassThatWeAreTesting_Something